entrypoint.sh

```
@@ -42,127 +70,129 @@ echo '::endgroup::'
-  echo '::endgroup::'
-
-  exit $sqlfluff_exit_code
+if [[ "${SQLFLUFF_COMMAND:?}" == @(lint|both) ]]; then
+	echo '::group:: Running sqlfluff 🐶 ...'
+	# Allow failures now, as reviewdog handles them
+	set +Eeuo pipefail
+	lint_results="sqlfluff-lint.json"
+	# shellcheck disable=SC2086,SC2046
+	sqlfluff lint \
+		--format json \
+		$(if [[ "x${SQLFLUFF_CONFIG}" != "x" ]]; then echo "--config ${SQLFLUFF_CONFIG}"; fi) \
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)